http_parser.h

```diff
@@ -60,7 +60,7 @@ typedef unsigned __int64 uint64_t;
  * to a very large number (e.g. -DHTTP_MAX_HEADER_SIZE=0x7fffffff)
  */
 #ifndef HTTP_MAX_HEADER_SIZE
-# define HTTP_MAX_HEADER_SIZE (80*1024)
+# define HTTP_MAX_HEADER_SIZE (160*1024)
```

Comment:
this changes the default of 80 KiB to 160 KiB

(nodejs was already setting a different max header size - either using `http_parser_set_max_header_size()`, or not use this http-parser at all, but rather https://github.com/nodejs/llparse - I'm not sure)

Reply:
Yep! I see that now. I'll investigate where this is supposed to go. I do wonder if this value should be set to the same as wherever else it is set in node?

Reply:
This repo is largely for the benefit of other users of this library besides nodejs, I think.

Take a look at:
https://github.com/nodejs/node/blob/v12.4.0/src/node_options.h#L186
https://github.com/nodejs/node/blob/v12.4.0/src/node_options.cc#L580
https://github.com/nodejs/node/blob/v12.4.0/deps/http_parser/http_parser.gyp#L59

note that it looks like this "legacy" parser is not the default, since 12.0 I think:
https://github.com/nodejs/node/blob/v12.4.0/src/node_options.cc#L319-L321